docs/document/website/versioned_docs/version-1.6.0/components/desktop/time-picker.md

```diff
@@ -0,0 +1,91 @@
+---
+id: version-1.6.0-time-picker
+title: TimePicker
+sidebar_label: TimePicker
+original_id: time-picker
+---
+
+## Overview
+
```

Comment:
Please create the same file with Japanese language at docs/document/website/translated_docs/ja/version-1.6.0